src/draw_parameters/mod.rs

```diff
@@ -882,3 +891,25 @@ fn sync_primitive_bounding_box(ctxt: &mut context::CommandContext,
         ctxt.state.primitive_bounding_box = value;
     }
 }
+
+fn sync_primitive_restart_index(ctxt: &mut context::CommandContext,
+                                enabled: bool)
+                                -> Result<(), DrawError>
+{
+    if ctxt.version >= &Version(Api::Gl, 3, 1)   || ctxt.version >= &Version(Api::GlEs, 3, 0) ||
```

Comment:
Please add a `TODO` comment saying to use `GL_PRIMITIVE_RESTART` if possible if `GL_PRIMITIVE_RESTART_FIXED_INDEX` is not supported.